core/src/main/java/com/linecorp/armeria/client/retry/RetryingClientBuilder.java

```diff
@@ -78,6 +78,9 @@ public T maxTotalAttempts(int maxTotalAttempts) {
      * dose not specify.
      *
      * @return {@link T} to support method chaining.
+     *
+     * @see <a href=https://line.github.io/armeria/advanced-retry.html#per-attempt-timeout>
+     *     Per-attempt-timeout</a>
```

Comment:
- There should be no whitespace between `>` and `Per`, because otherwise it will look like "[ this](https://line.github.io/armeria/)"
- `href` attribute value should be quoted.